code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -552,38 +555,48 @@
 		violence(user)
 
 /obj/item/healthanalyzer/simple/proc/greed_warning(mob/user)
-	to_chat(user, span_warning("\The [src] displays an eerily high-definition frowny face, chastizing you for asking it for too much encouragement."))
+	to_chat(user, span_warning("[src] displays an eerily high-definition frowny face, chastizing you for asking it for too much encouragement."))
 	show_emotion(AID_EMOTION_ANGRY)
 
 /obj/item/healthanalyzer/simple/proc/violence(mob/user)
 	playsound(src, 'sound/machines/buzz-sigh.ogg', 50, FALSE)
 	if(isliving(user))
 		var/mob/living/L = user
-		to_chat(L, span_warning("\The [src] makes a disappointed buzz and pricks your finger for being greedy. Ow!"))
+		to_chat(L, span_warning("[src] makes a disappointed buzz and pricks your finger for being greedy. Ow!"))
```

Comment:
we don't need this typecast anymore, could also be an early return as well

Reply:
this probably doesn't matter since it's out of scope but i just looked at [7bf109f](https://github.com/tgstation/tgstation/pull/79968/commits/7bf109f0241977891ea7b7b6923071c4ca566fdb) and noticed it

Reply:
I didn't really wanna do a whole rewrite of these simple analyzers since it's a tad out of scope for an attack chain pr